doc/conf.py

```diff
@@ -1012,35 +1012,27 @@ def reset_warnings(gallery_conf, fname):
 brand_icons = ('apple', 'linux', 'windows', 'discourse', 'python')
 fixed_width_icons = (
     # homepage:
-    'book', 'code-branch', 'newspaper', 'question-circle', 'quote-left',
+    'book', 'code-branch', 'newspaper', 'circle-question', 'quote-left',
     # contrib guide:
-    'bug', 'comment', 'hand-sparkles', 'magic', 'pencil-alt', 'remove-format',
-    'universal-access', 'discourse', 'python',
+    'bug-slash', 'comment', 'computer-mouse', 'hand-sparkles', 'pencil',
+    'text-slash', 'universal-access', 'wand-magic-sparkles',
+    'discourse', 'python',
 )
 other_icons = (
     'hand-paper', 'question', 'rocket', 'server', 'code', 'desktop',
-    'terminal', 'cloud-download-alt', 'wrench', 'hourglass'
+    'terminal', 'cloud-arrow-down', 'wrench', 'hourglass-half'
```

Comment:
name changes here reflect changes between FontAwesome 5 and 6.  This should hopefully make things more future-proof, as I think the old names go away with FA7.